modules/app-configuration/configuration-store/main.bicep

```diff
@@ -318,9 +318,11 @@ type privateEndpointType = {
   @description('Optional. A list of IP configurations of the private endpoint. This will be used to map to the First Party Service endpoints.')
   ipConfigurations: {
     name: string
-    groupId: string
-    memberName: string
-    privateIpAddress: string
+    properties: {
```

Comment:
The object has no metadata. Shoudl we maybe add some? I know it didn't do before, but I could swear I sawr you originally did :D

Reply:
Added :) .. updated readme and JSONs, and reran the pipelines above to validate